### DISCUSSION TITLE
Export Separate SRT file from a multi-Speaker file which has more than one speaker/composed in single channel

### DISCUSSION THREAD

**Discussion by Nimbusware** [deepgram/community#440] [General help] [answered]
Hi
would you help us how to get separate SRT files from a multi-Speaker file which has more than one speaker but composed in single channel by using API?

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @Nimbusware, we offer some pieces that you could put together to build this, but we don't offer it out-of-the-box. We offer diarization (`diarize=true`) to separate out different speakers. We also offer SRT output either using the [Python or Node SDKs](https://developers.deepgram.com/docs/automatically-generating-webvtt-and-srt-captions), or as a separate [JavaScript package](https://github.com/deepgram/deepgram-js-captions/). One way you could go about it would be to get the diarized transcription, post-process to create several subsets of the transcript JSON with isolated speakers, and then use our methods to get an SRT file for each of those individual-speaker JSON objects.

**Comment by Nimbusware** [comment]
@jkroll-deepgram thanks for your helpful reply
